templates/rapidsai/partials/clone_and_build_dask-sql.dockerfile.j2

```diff
@@ -0,0 +1,29 @@
+{# This partial clones the dask-sql repo and build and installs it into the rapids conda env. #}
+
+{# Install build prerequisites #}
+RUN gpuci_mamba_retry install -y -n rapids -c conda-forge \
+      "maven>=3.6.0" \
+      "pip" \
+      "setuptools_scm"
+
+{# Clone, build, install #}
+RUN mkdir -p ${DASK_SQL_DIR} \
+    && cd ${DASK_SQL_DIR} \
+    && git clone -b ${DASK_SQL_VER} https://github.com/dask-contrib/dask-sql dask-sql
+
+{# Add additional CUDA lib dir to LD_LIBRARY_PATH for "docker build".  This is
+not needed when using the nvidia runtime with "docker run" since the nvidia
+runtime also installs libcuda to a system location that client builds often
+find. #}
+
+{# Explicitly add the cuda runtime dir for the dask-sql build, then remove once
+build is done to keep the original LD_LIBRARY_PATH intact. #}
+ENV LD_LIBRARY_PATH_ORIG=${LD_LIBRARY_PATH}
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/cuda/compat
+
+RUN source activate rapids \
+    && cd ${DASK_SQL_DIR}/dask-sql \
+    && python -m pip install . --no-deps -vv
+
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH_ORIG}
+ENV LD_LIBRARY_PATH_ORIG=
```

Comment:
I think these `LD_LIBRARY_PATH` changes were all specific to Blazing. So we can probably try removing all references at this point

Reply:
Done